package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "standard 'src/**/*.js' 'test/**/*.js' storages.js constants.js && BABEL_ENV=commonjs ava",
     "test:watch": "npm test -- --watch",
-    "copy-types": "cp -r type-definitions/ lib/",
-    "clean": "rimraf lib dist es",
+    "copy-types": "rsync -av --exclude=constants.d.ts --exclude=storages.d.ts type-definitions/ lib/ && cp type-definitions/constants.d.ts type-definitions/storages.d.ts ./",
```

Comment:
having to colocate type files is a messy unforeseen consequence of putting storages and constants in root :(

In particular for storages we do it to help with dead code elimination for projects that do not use the default storage.

is there no other way to tell typescript where to look for these definitions?